aleph/search/__init__.py

```diff
@@ -103,6 +103,11 @@ def get_index(self):
         return xref_index()
 
 
+class XrefPartyQuery(XrefQuery):
```

Comment:
The alternative to subclassing here would be to define an option in ``XrefQuery.SORT_FIELDS`` which makes the sort available via the normal API.

Reply:
magic. although then i'd have to modify the request options parser so i can create one XrefQuery with the evaluation mode on and another with it off... right?